helm.el

```diff
@@ -1,5 +1,8 @@
 ;;; helm.el --- Emacs incremental and narrowing framework -*- lexical-binding: t -*-
 
+;; Version: 1.0
```

Comment:
required for Cask

Reply:
Why not putting the right number version e.g. 3.5.6 which is the next version number should be released soon now)?